### PR TITLE
has payu been added to payment processors

### PR DESCRIPTION
**Pull request by Alec4r**
has been added test flag to the payment processor of payu

### CODE REVIEW

File: ecommerce/extensions/payment/views/payu.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            site_configuration=basket.site.siteconfiguration
+        )
+
+        try:
+            with transaction.atomic():
+                try:
+                    self.handle_payment(payu_response, basket)
+                except PaymentError:
+                    logger.exception('PaymentError for basket [%d] failed.', basket.id)
+                    if transactionState == '7':
```
Comment (jfavellar90) [inline ecommerce/extensions/payment/views/payu.py]: This is fucking terrible, can you add a comment above this line as a TODO to investigate why the value '7'?

File: ecommerce/extensions/payment/views/payu.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
```
Comment (jfavellar90) [inline ecommerce/extensions/payment/views/payu.py]: I assume we didn't apply changes on this file (which have not nice things) but we can review later

File: ecommerce/extensions/payment/processors/payu.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        return wfirma
+
+    def _verify_student(self, site, username):
+        import pudb; pu.db
+        path_api = "/api/user/v1/preferences/{}/change_to_verified_mode/".format(username)
+        url = get_lms_url(path_api)
+        access_token = site.siteconfiguration.access_token
+
+        headers = {
+            'authorization': "JWT {}".format(access_token),
```
[the remark this answers is not in the document]
Comment (Alec4r) [reply ecommerce/extensions/payment/processors/payu.py]: using a site method that returns the token

File: ecommerce/extensions/payment/views/payu.py (hunk 1 above)
Comment (Alec4r) [reply ecommerce/extensions/payment/views/payu.py]: in the previous comments said that it meant transactionState

**Pull request merged**